package.json

```diff
@@ -62,7 +62,7 @@
     "coveralls": "^3.0.2",
     "function-bind": "^1.0.2",
     "karma": "^3.0.0",
-    "karma-browserify": "^5.0.1",
+    "karma-browserify": "^8.0.0",
```

Comment:
The peer dependencies of `karma-browserify@8.0.0` are `"karma": ">=4.3.0"`.

The graphite app did not explain why it thought that `karma@6` is required. It could be true, but we don't know why.